File: problem-solving/Easy/7_mini-max-sum/javascript/main.js
// URL : https://www.hackerrank.com/challenges/mini-max-sum/problem

function miniMaxSum(arr) {
  let mini = 0,
    max = 0;
  mini = [...arr]
    .sort()
    .splice(0, 4)
    .reduce((acc, value) => acc + value, 0);
  max = [...arr]
    .sort()
    .reverse()
    .splice(0, 4)
    .reduce((acc, value) => acc + value, 0);
  console.log(mini, max);
}

// test
miniMaxSum([1, 2, 3, 4, 5]);

File: problem-solving/Easy/7_mini-max-sum/python/main.py
# URL : https://www.hackerrank.com/challenges/mini-max-sum/problem

def miniMaxSum(arr):
    mini = sum(sorted(arr)[0:4])
    max = sum(sorted(arr, reverse=True)[0:4])
    print(mini, max)


# test
miniMaxSum([3, 5, 1, 9, 7])
